config/environments/folio_sync_dev.rb

```diff
@@ -4,4 +4,5 @@
 
 Rails.application.configure do
   config.log_level = :debug
+  config.action_mailer.default_url_options = { host: 'diglib-rails-dev-1.cul.columbia.edu' }
```

Comment:
```suggestion
  config.action_mailer.default_url_options = { host: 'diglib-rails-dev1.cul.columbia.edu' }
```
Sorry, didn't spot this when we were meeting earlier!

Also, I'd recommend adding configurations in folio_sync_test.rb and folio_sync_prod.rb with similar configurations (just swapping in test1 and prod1 in those cases).